package.json

```diff
@@ -21,7 +33,7 @@
   },
   "scripts": {
     "start": "vite build --watch",
-    "build": "tsc && vite build",
```

Comment:
Don't actually know if we need tsc here as the plugin generates the types for us 🤔